src/scripts/heroAnimation/__tests__/utils.test.js

```diff
@@ -126,10 +126,20 @@ describe("drawLines function", () => {
 });
 
 describe("getDistance function", () => {
-  it("should return distance between 2 points", () => {
+  it("should calculate distance between 2 points using hypot as default method", () => {
     const distance = utils.getDistance(point1, point2);
 
-    expect(distance).toEqual(200);
+    expect(distance).toEqual(141.4213562373095);
```

Comment:
perhaps something that can be improved in subsequent iterations, but shouldn't we round up to the second decimal place instead?